contrib/completion/bash/docker

```diff
@@ -70,6 +85,57 @@ __docker_container_ids() {
 	COMPREPLY=( $(compgen -W "${containers[*]}" -- "$cur") )
 }
 
+__docker_images() {
+	local images_args=""
+	case "$DOCKER_COMPLETION_SHOW_IMAGE_IDS" in
+		all)
+			images_args="--no-trunc -a"
+			;;
+		non-intermediate)
+			images_args="--no-trunc"
+			;;
+	esac
+
+	local awk_script=''
+
+	case "$DOCKER_COMPLETION_SHOW_IMAGE_ID" in
```

Comment:
Should be `$DOCKER_COMPLETION_SHOW_IMAGE_IDS`.
Currently `DOCKER_COMPLETION_SHOW_IMAGE_IDS` has no effect at all.